test/acceptance/user-order.controller.acceptance.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {supertest, expect} from '@loopback/testlab';
+import {ShoppingApplication} from '../..';
+import {OrderRepository, UserRepository} from '../../src/repositories';
+import {MongoDataSource} from '../../src/datasources';
+import {User, Order} from '../../src/models';
+import {setupApplication} from './helper';
+
+describe('UserOrderController acceptance tests', () => {
+  let app: ShoppingApplication;
+  let client: supertest.SuperTest<supertest.Test>;
```

Comment:
Ditto, use `Client` instead of `supertest.SuperTest<supertest.Test>`.

Reply:
I fixed that with #16